staging/src/k8s.io/code-generator/_examples/HyphenGroup/listers/example/v1/testtype.go

```diff
@@ -83,7 +83,11 @@ func (s testTypeNamespaceLister) List(selector labels.Selector) (ret []*v1.TestT
 
 // Get retrieves the TestType from the indexer for a given namespace and name.
 func (s testTypeNamespaceLister) Get(name string) (*v1.TestType, error) {
```

Comment:
TestType is a namespaced type (see `// +genclient` comment in the types.go file, rather than a `// +genclient:nonNamespaced` comment for `ClusterTestType`). That means that getting an object with no namespace is not valid and is expected to return an error